### DISCUSSION TITLE
Virtualbox CentOS 7 not available for dropdown

### DISCUSSION THREAD

**Discussion by waitesgithub** [Security-Onion-Solutions/securityonion#6365] [Unsupported Versions]
I have tried to install Security Onion 2.3.90 several times on Virtualbox and not able to get it installed. I feel the issue is that Virtualbox does not have CentOS 7 available for download at this time as a dropdown option to select for the installation per your documentation. I will try to use a CentOS 7 image and download SecurityOnion on top of that to see if this will resolve the issue with installation.

**Comment by tomjones1977** [comment]
It would help to tell us what actually goes wrong when you try to install from the ISO.  I have installed SO in VirtualBox several times without issue.

What happens when you try to install - how far into the install do you get, what error messages appear, what settings have you given the VM?

**Reply by iqworks** [reply]
tomjones, can you tell us what version you used besides centos?

**Reply by tomjones1977** [reply]
@iqworks I'm not sure what you mean.  If you're talking about the VM type to choose during creation, the Other Linux option should be fine.
